langs/nl.json

```diff
@@ -43,7 +43,7 @@
         "add": {
             "addNew": "Voeg {category} toe",
             "addNewMapLabel": "Klik hier om een item toe te voegen",
-            "confirmButton": "Voeg een {category} toe<br/><div class='alert'>Je toevoeging is voor iedereen zichtbaar</div>",
+            "confirmButton": "Voeg een {category} toe<br><div class=\"alert\">Je toevoeging is voor iedereen zichtbaar</div>",
```

Comment:
Doesn't this break the CSS?

Reply:
This is fine. Also: translators don't see this here

Reply:
Oh I know, it just looked pretty weird to me. I'll assume Weblate knows what it is doing then :D